### ISSUE TITLE
Mod does not respect frozen perishable stats

### ISSUE THREAD

**Issue by Tharn** [emipa606/BestMix#3]
As the title says. Got a big freezer with dozens of different meats. In frozen state, it consistently selects Ibex Meat (spoils in 1.6 days), yet when I turn the freezer off and look at the actual data, there are many meat stacks ready to spoil within hours.

When the freezer is off, it correctly selects the meats that perish first. But then I can't use the freezer. Defeats the purpose of the whole thing.

**Comment by emipa606** [comment]
That is not really what the setting checks for. It checks how long until the meat spoils at its current temperature. Frozen items never spoil.

**Comment by Tharn** [comment]
Yeah but they do have a spoilage timer, so every blackout turns into a spoil-fest. Being able to get the short-expiry foods out of the way first would still help, but if it's technically not feasible I understand.
